tests/test_api.py

```diff
@@ -34,3 +34,22 @@ def tracer(self):
     def test_binary_propagation(self):
         # TODO binary codecs are not implemented at the moment
         pass
+
+    def is_parent(self, parent, span):
+        return span.parent_id == getattr(parent, 'span_id', None)
+
+    # NOTE: this overrides a method defined in ApiComaptibilityCheckMixin in
+    # order to add the scope.close() call, which prevents the tracer from
+    # leaving an active scope in thread local storage
+    # TODO: remove after
+    # https://github.com/opentracing/opentracing-python/issues/117 is fixed
```

Comment:
it looks like the code is already fixed, but we just need a new release

Reply:
You're right, any idea when is the `2.1.0` coming? If it's not coming anytime soon then I'll leave the override and change the `TODO` "remove after moving to 2.1.0 of python-opentracing"

Reply:
I'll try to get a release in the next few days. Meanwhile a better ticket to reference is https://github.com/opentracing/opentracing-python/issues/119

Reply:
done